src/internal/analytics/components/analytics-funnel.tsx

```diff
@@ -157,6 +157,12 @@ const InnerAnalyticsFunnel = ({ mounted = true, children, stepConfiguration, ...
         },
       ];
 
+      let componentTheme = THEME;
+      if (THEME === 'polaris') {
+        // This is the only place we specify the theme as classic so we cannot reuse the getVisualTheme function :(
```

Comment:
Can we talk to the consumers of this code and ask them to stop relying on this?

Also how much is there real data with that field? According to my data, it is a very small number

Reply:
Yes this was my plan after this. I need to dive into where they got used but for now just maintaining the existing functionality.